matrix/server.py

```diff
@@ -671,9 +671,12 @@ def schedule_reconnect(self):
         self.reconnect_time = time.time()
 
         if self.reconnect_delay:
-            self.reconnect_delay = self.reconnect_delay * 2
+            self.reconnect_delay = self.reconnect_delay *
```

Comment:
This is a syntax error, you can put the right hand side into parentheses to get around having the whole thing on the same line.

Reply:
I make installed and tested this… how?

Anyway, I tend to use \, so not sure which style you'd like.